System.Compression.Primitives/System/IO/Compression/Brotli/BrotliStream.cs

```diff
@@ -10,20 +10,18 @@
 #else
 using nuint = System.UInt32;
 #endif 
-namespace System.IO.Compression.Brotli
+namespace System.IO.Compression
 {
     public partial class BrotliStream : Stream
     {
 
         private const int DefaultBufferSize = 64 * 1024;
         private int BufferSize;
-        private Stream _stream;
+        public Stream _stream;
         private CompressionMode _mode;
         private nuint TotalOut { get; }
-        //private nuint AvailOut;
         private IntPtr AvailOut;
```

Comment:
There is inconsistency in the variable names.
Change so all private variable names start with underscore and lower case (following camel casing).

> 3. We use `_camelCase` for internal and private fields and use `readonly` where possible. Prefix instance fields with `_`, static fields with `s_` and thread static fields with `t_`. When used on static fields, `readonly` should come after `static` (i.e. `static readonly` not `readonly static`).